components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -276,6 +279,8 @@ be lost if SCREAM_HACK_XML is not enabled.
 
     <mac_aero_mic inherit="atm_proc_group">
       <atm_procs_list>(shoc,cldFraction,spa,p3)</atm_procs_list>
+      <atm_procs_list hgrid=".*pg2">(tms,shoc,cldFraction,spa,p3)</atm_procs_list> <!-- TMS only available for PG2 -->
+      <atm_procs_list COMPSET=".*SCREAM.*noAero" hgrid=".*pg2">(tms,shoc,cldFraction,p3)</atm_procs_list>
```

Comment:
@jgfouca may correct me, but I think this should come _after_ the next line. CIME should use the last matching entry, so you want the most specific ones after the general one.

Reply:
Agree.